code/build/template/data/config/[=name].ini

```diff
@@ -26,6 +26,27 @@ FrustumHeight   = 720
 FrustumFar      = 2
 FrustumNear     = 0
 Position        = (0, 0, -1) ; Objects with -1 <= Z <= 1 will be visible
+[+inspector
+
+[OrxInspector]
+Input          = OrxInspectorInput
+ObjectShader   = OrxInspectorShader
+
+[OrxInspectorInput]
+KEY_I           = Pick

```

Comment:
Hovering over an object with a mouse pointer and pressing `I` on the keyboard will open an inspector window for that object. The input set is enabled and disabled at `Init` and `Exit` times, respectively.

Reply:
Given it requires mouse picking, I'd go with a mouse input instead. What do you think?

Reply:
I started with middle click as you'd suggested. I moved to a keyboard shortcut in its place as it made it easier for me to use on a laptop. What do you think of including both?

Reply:
Sounds good to me!